### PR TITLE
[1534] Added support for .woff and more

### PR DESCRIPTION
**Pull request by stanica**

### CODE REVIEW

File: css/webfonts.less [hunk 1]
```diff
@@ -3,27 +3,103 @@
 */
 
 @font-face {
-  font-family: 'Open Sans';
-  font-style: italic;
-  font-weight: 400;
-  src: local('Open Sans Italic'), local('Open-Sans-Italic'),
-    url('../resources/fonts/OpenSans-Italic.ttf') format('truetype');
+    font-family: 'OpenSansCondensedLightRegular';
```
Comment (k88hudson) [inline css/webfonts.less]: Remove, not needed

File: css/webfonts.less [hunk 2]
```diff
@@ -3,27 +3,103 @@
...
+    font-style: normal;
 }
 
 @font-face {
-  font-family: 'Open Sans';
-  font-style: normal;
-  font-weight: 700;
-  src: local('Open Sans Bold'), local('Open-Sans-Bold'),
-    url('../resources/fonts/OpenSans-Bold.ttf') format('truetype');
+    font-family: 'OpenSansCondensedLightItalic';
```
Comment (k88hudson) [inline css/webfonts.less]: Remove

File: css/webfonts.less [hunk 3]
```diff
@@ -3,27 +3,103 @@
...
+         url('opensans-condlightitalic-webfont.woff') format('woff'),
+         url('opensans-condlightitalic-webfont.ttf') format('truetype'),
+         url('opensans-condlightitalic-webfont.svg#OpenSansCondensedLightItalic') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+
+@font-face {
+    font-family: 'OpenSansBold';
```
Comment (k88hudson) [inline css/webfonts.less]: font-family: "Open Sans"

File: css/webfonts.less [hunk 4]
```diff
@@ -3,27 +3,103 @@
...
+    src: url('opensans-bold-webfont.eot');
+    src: url('opensans-bold-webfont.eot?#iefix') format('embedded-opentype'),
+         url('opensans-bold-webfont.woff') format('woff'),
+         url('opensans-bold-webfont.ttf') format('truetype'),
+         url('opensans-bold-webfont.svg#OpenSansBold') format('svg');
+    font-weight: normal;
```
Comment (k88hudson) [inline css/webfonts.less]: font-weight: 700

File: css/webfonts.less [hunk 5]
```diff
@@ -3,27 +3,103 @@
...
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'OpenSansBoldItalic';
```
Comment (k88hudson) [inline css/webfonts.less]: Open Sans

File: css/webfonts.less [hunk 6]
```diff
@@ -3,27 +3,103 @@
...
+    src: url('opensans-bolditalic-webfont.eot');
+    src: url('opensans-bolditalic-webfont.eot?#iefix') format('embedded-opentype'),
+         url('opensans-bolditalic-webfont.woff') format('woff'),
+         url('opensans-bolditalic-webfont.ttf') format('truetype'),
+         url('opensans-bolditalic-webfont.svg#OpenSansBoldItalic') format('svg');
+    font-weight: normal;
```
Comment (k88hudson) [inline css/webfonts.less]: 700

File: css/webfonts.less [hunk 7]
```diff
@@ -3,27 +3,103 @@
...
+    font-style: normal;
```
Comment (k88hudson) [inline css/webfonts.less]: italic

File: css/webfonts.less [hunk 8]
```diff
@@ -3,27 +3,103 @@
...
+}
+
+@font-face {
+    font-family: 'OpenSansExtraboldRegular';
```
Comment (k88hudson) [inline css/webfonts.less]: remove

File: css/webfonts.less [hunk 9]
```diff
@@ -3,27 +3,103 @@
...
+    src: url('opensans-extrabold-webfont.eot?#iefix') format('embedded-opentype'),
+         url('opensans-extrabold-webfont.woff') format('woff'),
+         url('opensans-extrabold-webfont.ttf') format('truetype'),
+         url('opensans-extrabold-webfont.svg#OpenSansExtraboldRegular') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'OpenSansExtraboldItalic';
```
Comment (k88hudson) [inline css/webfonts.less]: remove

File: css/webfonts.less [hunk 10]
```diff
@@ -3,27 +3,103 @@
...
+    src: url('opensans-extrabolditalic-webfont.eot?#iefix') format('embedded-opentype'),
+         url('opensans-extrabolditalic-webfont.woff') format('woff'),
+         url('opensans-extrabolditalic-webfont.ttf') format('truetype'),
+         url('opensans-extrabolditalic-webfont.svg#OpenSansExtraboldItalic') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'OpenSansRegular';
```
Comment (k88hudson) [inline css/webfonts.less]: Open Sans

File: css/webfonts.less [hunk 11]
```diff
@@ -3,27 +3,103 @@
...
+    src: url('opensans-regular-webfont.eot');
+    src: url('opensans-regular-webfont.eot?#iefix') format('embedded-opentype'),
+         url('opensans-regular-webfont.woff') format('woff'),
+         url('opensans-regular-webfont.ttf') format('truetype'),
+         url('opensans-regular-webfont.svg#OpenSansRegular') format('svg');
+    font-weight: normal;
+    font-style: normal;
```
Comment (k88hudson) [inline css/webfonts.less]: you can remove both font-weight and font-style

File: css/webfonts.less [hunk 12]
```diff
@@ -3,27 +3,103 @@
...
+
+@font-face {
+    font-family: 'OpenSansItalic';
+    src: url('opensans-italic-webfont.eot');
+    src: url('opensans-italic-webfont.eot?#iefix') format('embedded-opentype'),
+         url('opensans-italic-webfont.woff') format('woff'),
+         url('opensans-italic-webfont.ttf') format('truetype'),
+         url('opensans-italic-webfont.svg#OpenSansItalic') format('svg');
+    font-weight: normal;
+    font-style: normal;
```
Comment (k88hudson) [inline css/webfonts.less]: Italic

File: css/webfonts.less [hunk 13]
```diff
@@ -3,27 +3,103 @@
...
+    src: url('opensans-regular-webfont.eot?#iefix') format('embedded-opentype'),
+         url('opensans-regular-webfont.woff') format('woff'),
+         url('opensans-regular-webfont.ttf') format('truetype'),
+         url('opensans-regular-webfont.svg#OpenSansRegular') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'OpenSansItalic';
```
Comment (k88hudson) [inline css/webfonts.less]: Open Sans

File: css/webfonts.less [hunk 14]
```diff
@@ -3,27 +3,103 @@
...
+    font-style: normal;
 }
 
 @font-face {
-  font-family: 'Open Sans';
-  font-style: normal;
-  font-weight: 400;
-  src: local('Open Sans Regular'), local('Open-Sans-Regular'),
-    url('../resources/fonts/OpenSans-Regular.ttf') format('truetype');
+    font-family: 'OpenSansLightItalic';
```
Comment (k88hudson) [inline css/webfonts.less]: remove

**Comment by jbuck** [comment]
Closing in favour of pull request #727

**Pull request closed without merging**